readJSON.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
You can do better than this. Use mongodb for a simpler and more reliable code base.

Reply:
Will get to this!